trollmoves/movers.py

```diff
@@ -201,10 +209,30 @@ class FtpMover(Mover):
     active_connections = dict()
     active_connection_lock = Lock()
 
+    # Check if usernams, password is stored in a netrc file
+
+    def _get_netrc_authentication(self):
+        """Get login authentications from netrc file if available"""
+        if NETRCFILE and os.path.exists(NETRCFILE):
+            try:
+                secrets = netrc.netrc(NETRCFILE)
+            except netrc.NetrcParseError:
+                logger.warning('Failed retrieve authentification details from netrc file!')
+                return
+
+            secrets = netrc.netrc(NETRCFILE)
+            if self.destination.hostname in secrets.hosts:
+                self.destination.username, account, self.destination.password = secrets.authenticators(
+                    self.destination.hostname)
+                logger.debug('Got username and password from netrc file!')
+
     def open_connection(self):
+        """Open the connection and login."""
         connection = FTP(timeout=10)
         connection.connect(self.destination.hostname,
                            self.destination.port or 21)
+
+        self._get_netrc_authentication()
```

Comment:
This should be moved a couple of lines further down, so as not to overwrite username and password if they were provided in `destination`

Reply:
Ok. Will do. Please note that _get_netrc_authentication only change the username and password if the netrc file is available and the hostname is included there.